.github/workflows/build_push_service.yml

```diff
@@ -6,20 +6,169 @@ name: sc-app-build-push
 # A workflow run is made up of one or more jobs that can run sequentially or in parallel
 jobs:
-  # This workflow contains a single job called "deploy"
-  build:
+  # This workflow contains a single job called "build_and_push"
+  build_and_push:
     # The type of runner that the job will run on
     runs-on: ubuntu-latest
     env:
       # Setting environment variables here make them available for all steps
-      CI_GITHUB_TRAVISUSER_SWARMVM_KEY: ${{ secrets.CI_GITHUB_TRAVISUSER_SWARMVM_KEY }}
-      DOCKER_USERNAME: ${{ secrets.DOCKER_USERNAME }}
-      DOCKER_TOKEN: ${{ secrets.DOCKER_TOKEN }}
+      AVSCANFILE_REPO: schulcloud-avcheck-scanfile
+      AVSCANURL_REPO: schulcloud-avcheck-scanurl
+      AVSCANWEBSERVER_REPO: schulcloud-avcheck-scanurl
+      DOCKER_NAMESPACE: schulcloud
+      BRANCH_NAME: ""
+      AVCHECK_VERSION: ""
+      DOCKER_TAG_POSTFIX: ""
+      DOCKER_TAG_SHA: ""
+      JIRA_TicketID: ""
+      BRANCH_PREFIX: ""
 
     # Steps represent a sequence of tasks that will be executed as part of the job
     steps:
       # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it must be on first place to allow central setting of working directory
       - uses: actions/checkout@v2
+      - name: Get Branch name
+        id: branch
+        uses: nelonoel/branch-name@v1.0.1
+      - name: Set BRANCH_NAME
+        run: echo Set BRANCH_NAME
+        env:
+          BRANCH_NAME: ${{ steps.branch.outputs.BRANCH_NAME }}
+      - name: Read version file
+        # The version to be used for tagging the images is read out of the version file
+        id: read_version
+        uses: juliangruber/read-file-action@v1
+        with:
+          path: ./version
+      - name: Set env AVCHECK_VERSION
+        run: |
+          echo "Version of antivirus_service is ${{ steps.read_version.outputs.content }}"
+          echo "AVCHECK_VERSION=${{ steps.read_version.outputs.content }}" >> $GITHUB_ENV
+      - name: RegEx JIRA ticket ID
+        uses: actions-ecosystem/action-regex-match@v2
+        id: jira
+        with:
+          text: ${{ env.BRANCH_NAME }}
+          regex: '^hotfix|feature\/([A-Z]+-[0-9]+)-[a-zA-Z_]+$'
+          flags: gm
+      - name: Set env var JIRA_TicketID
+        run: |
+          echo "JIRA_TicketID is ${{ steps.jira.outputs.group1 }}"
+          echo JIRA_TicketID=${{ steps.jira.outputs.group1 }} >> $GITHUB_ENV
+      # Assemble the Dockertag for the build image based on the type of the branch
+      - name: Find JIRA ticket BRANCH_PREFIX
+        uses: actions-ecosystem/action-regex-match@v2
+        id: branch_prefix
+        with:
+          text: ${{ env.BRANCH_NAME }}
+          regex: '^([a-zA-Z]+)\/[A-Z]+-[0-9]+-[a-zA-Z_]+$'
+          flags: gm
+      - name: Set env var JIRA BRANCH_PREFIX
+        run: |
+          echo "BRANCH_PREFIX is ${{ steps.branch_prefix.outputs.group1 }}"
+          echo BRANCH_PREFIX=${{ steps.branch_prefix.outputs.group1 }} >> $GITHUB_ENV
```

Comment:
Why is it put to ENV vars if is at the output?

Reply:
So other actions have access to env vars, we need this var to build correct dockertag with new naming convention.

Reply:
okay, then my question where not so good to understand.
Why do you put the output form an named step to the env vars again and not uesed the step output later?
The steps.branch_prefix.outputs.group1 is allrady an env var.....